cli/cmd/plugin/ui/web/tanzu-ui/README.md

```diff
@@ -85,3 +85,10 @@ If your Tanzu CLI and the UI plugin are installed correctly, running `tanzu ui`
 [Code/File formatting best practices](./FORMATTING.md)
 
 [REST API Consumption](./RESTAPIS.md)
+
+## Build Contextual Help Docs
+
+HTMl Docs are located at `src/assets/contextualHelpDocs`.\
+Run `npm run build-index`.\
```

Comment:
I think it'd be nice to have this spelled out in a little more detail.
As in: `To XXX, run yyy`. Perhaps something like "To generate the docs index (which is a prerequisite for contextual help, run npm run build-index. This should be done every time a new topic is introduced.`
I'm also unclear as to what the "Fuse Index" is.

Reply:
updated